SourceryTests/Parsing/ComposerSpec.swift

```diff
@@ -151,7 +151,7 @@ class ParserComposerSpec: QuickSpec {
                                 TupleElement(name: "2", typeName: TypeName("String")),
                                 TupleElement(name: "3", typeName: TypeName("Float")),
                                 TupleElement(name: "literal", typeName: TypeName("[String: [String: Float]]", dictionary: DictionaryType(name: "[String: [String: Float]]", valueTypeName: TypeName("[String: Float]", dictionary: DictionaryType(name: "[String: Float]", valueTypeName: TypeName("Float"), keyTypeName: TypeName("String"))), keyTypeName: TypeName("String")))),
-                                TupleElement(name: "generic", typeName: TypeName("Dictionary<String, Dictionary<String, Float>>", dictionary: DictionaryType(name: "Dictionary<String, Dictionary<String, Float>>", valueTypeName: TypeName("Dictionary<String, Float>", dictionary: DictionaryType(name: "Dictionary<String, Float>", valueTypeName: TypeName("Float"), keyTypeName: TypeName("String"))), keyTypeName: TypeName("String")))),
+                                TupleElement(name: "generic", typeName: TypeName("Dictionary<String, Dictionary<String, Float>>", generic: GenericType(name: "Dictionary", referencedTypeNames: [TypeName("String"), TypeName("Dictionary<String, Float>", generic: GenericType(name: "Dictionary", referencedTypeNames: [TypeName("String"), TypeName("Float")]))]), dictionary: DictionaryType(name: "Dictionary<String, Dictionary<String, Float>>", valueTypeName: TypeName("Dictionary<String, Float>", generic: GenericType(name: "Dictionary", referencedTypeNames: [TypeName("String"), TypeName("Float")]), dictionary: DictionaryType(name: "Dictionary<String, Float>", valueTypeName: TypeName("Float"), keyTypeName: TypeName("String"))), keyTypeName: TypeName("String")))),
```

Comment:
You don't need to scan anything for these tests, just create types programmatically and pass them to `Composer`. There are tests like that (to test types from different modules) in `ComposerSpecs`